explorer/interpreter/interpreter.cpp

```diff
@@ -1684,7 +1685,7 @@ auto Interpreter::StepStmt() -> ErrorOr<Success> {
             std::make_unique<ExpressionAction>(&cast<For>(stmt).loop_target()));
       }
       if (act.pos() == 1) {
-        Nonnull<const TupleValue*> source_array =
+        auto source_array =
             cast<const TupleValue>(act.results()[TargetVarPosInResult]);
```

Comment:
```suggestion
        const auto* source_array =
            cast<TupleValue>(act.results()[TargetVarPosInResult]);
```
... seems more idiomatic to me.